test/chpldoc/module/ordering.good

```diff
@@ -1,6 +1,6 @@
 Module: ordering
-   config var first: string
-      This config var should be printed first 
    proc second()
       This proc should be printed after all config vars 
 
+   config var first: string
+      This config var should be printed first 
```

Comment:
Likewise, here the comments are no longer accurate.  Sorry for writing comments/tests like this :(